processors/sshd/sshdProcessor.go

```diff
@@ -67,6 +69,52 @@ func SetLogger(l *zap.SugaredLogger) {
 	logger = l
 }
 
+type syslogMessage struct {
+	PID     string
+	Message string
+}
+
+func (s *SshdProcessor) Process(ctx context.Context, r io.Reader, currentLog *bytes.Buffer, buf []byte) (int, error) {
+	n, err := r.Read(buf[:cap(buf)])
```

Comment:
this func reads in bytes from the namedpipe ingester and calls `ProcessEntry` on every newline.